samples/host_verify/README.md

```diff
@@ -0,0 +1,68 @@
+
+# The host verification sample
+
+- Demonstrates the attestation of a remote host
+- Requires only the `openenclave/host_verify.h` header, as well as some C standard header files
+- Also runs in simulation mode and when both BUILD_ENCLAVES and HAS_QUOTE_PROVIDER flags are turned off
+
+Prerequisites: you may want to read [Common Sample Information](../README.md#common-sample-information) before going further
+
+## About the host verification sample
+
+When a host is doing host verification, or remote host attestation to be exact, it means that it is trying to authenticate its hardware and software settings so that it can determine whether or not to trust the remote host.
```

Comment:
It would be useful somewhere (maybe in another doc that one can point to?) to explain how such a trust decision is made.   https://tools.ietf.org/html/draft-ietf-rats-architecture explains that Appraisal Policy for Attestation Results can control whether an attestation result is considered trusted or not, but I don't think this sample uses any such policy other than whether an attestation result was generated or not, correct?   Would be useful to say that somewhere since in general someone that wants to copy this sample into a real scenario may need to extend it as such.